instructor/common.py

```diff
@@ -261,10 +261,6 @@ def assignments_variables(request):
     if request.method == "POST":
         form = AssignmentForm(request.POST, instance=assignment)
         if form.is_valid():
-            # If form has changed want to delete existing Treatments
```

Comment:
This is exactly what we discussed with Alison today. Do you think deleting and recreating all objects is a good idea?That would be the easiest :). Otherwise, I could set the corresponding field to null and  remove all duplicates. But this would work only if I am unselecting variables.